fabricadesoftware-ifc/fabricaNode-BackEnd
# FEAT: Adding Scripts

### Adding scripts to reduce difficulties

**Description:**
This issue adding scripts to reduce the difficulties on the code

**Test:**

1. execute "pdm run populate" on terminal
2. open the server
3. access the endpoint: "/api"
4. access all endpoints on /api and verify if all tables have data

**Have you fulfilled the entire issue description?**
